pos_operating_unit/report/pos_order_report.py

```diff
@@ -0,0 +1,15 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
I would remove this header. The other files don't contain it

Reply:
Yeap, this header is probably copy pasted from upstream odoo/odoo code.
Could you remove it?